### DISCUSSION TITLE
For you

### DISCUSSION THREAD

**Discussion by rolivegab** [community/community#13259] [Other Feature Feedback, Questions, & Ideas]
I really want to have some option to not see someone's updates.
have people here that cause a lot of bad things to me, and my "For you" section gives me a hard bad feelings.

**Comment by lucapette** [comment]
I came here to say the same thing.

**Comment by kuncgregor** [comment]
just why?

**Comment by ab22593k** [comment]
Great feature 💯

**Comment by jfullstackdev** [comment]
hey guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
